MailCore/API/MailApiFetcher.swift

```diff
@@ -291,18 +291,19 @@ public class MailApiFetcher: ApiFetcher {
 }
 
 class SyncedAuthenticator: OAuthAuthenticator {
-    @InjectService var networkLoginService: InfomaniakLogin
-
     override func refresh(
         _ credential: OAuthAuthenticator.Credential,
         for session: Session,
         completion: @escaping (Result<OAuthAuthenticator.Credential, Error>) -> Void
     ) {
         AccountManager.instance.refreshTokenLockedQueue.async {
+            @InjectService var keychainHelper: KeychainHelper
```

Comment:
As they are servicies, and are not subject to change identity, it might be better to keep them as lazy iVars(LazyInjectService).

That way we do not re-do the resolution each time we perform a refresh.

Reply:
I don't want to reference self